composer.json

```diff
@@ -4,27 +4,46 @@
     "description": "Sitemap Plugin for Sylius",
     "license": "MIT",
     "require": {
-        "php": "^7.1",
-        "sylius/sylius": "^1.0"
+        "php": "^7.2",
+        "sylius/sylius": "^1.3"
     },
     "require-dev": {
-        "lchrusciel/api-test-case": "^2.0|^3.1.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^2.0|^3.0",
-        "phpspec/phpspec": "^4.0|^5.0",
-        "phpstan/phpstan": "^0.11.1",
-        "phpunit/phpunit": "^6.0|^7.0",
-        "sylius-labs/coding-standard": "^3.0"
+        "lchrusciel/api-test-case": "^4.0",
```

Comment:
Previous comment also applies here probably. For some reason I like to keep users attached by not forcing them continuously to upgrade. WDYT?

Reply:
For dev requires I do not agree. Development requirements should just be easy for you in my opinion.